code/src/Pages/Home.jsx

```diff
@@ -0,0 +1,58 @@
+import React, {useState, useEffect} from "react";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import LoadingSpinner from "Components/LoadingSpinner";
```

Comment:
Here you can add space between dependencies you download and the components you import according to Technigo's convention.